apps/web/components/ui/HeroCarousel/types.ts

```diff
@@ -7,7 +7,7 @@ export type SizeKey = 'lg' | 'md' | 'sm' | 'xs';
 
 export type Sizes = Record<SizeKey, { width: string; height: string }>;
 
-export type HeroItem = {
+export type HeroContentProps = {
   image?: string;
```

Comment:
**question**: Don't we still have three sizes here?

Reply:
yes, and the type reflects that. it comes as a lg, md, sm or xs

Reply:
Oh I didn't see it down there, sorry.